packages/bbot/data_stream/asm_intel/fields/fields.yml

```
@@ -0,0 +1,246 @@
+- name: bbot
+  type: group
+  fields:
+    - name: data.dns_name
+      type: keyword
+      description: >
+        dns name found
```

Equivalent throughout …
```suggestion
        DNS name found.
```